faqs/galaxy/analysis_reporting_issues.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Reporting Issues and Feature Requests
+area: analysis
+box_type: tip
+layout: faq
+contributors: [jennaj, garimavs]
+---
+
+- For reporting **Usage Problems**, related to tools and functions, head to the [Galaxy Help](https://help.galaxyproject.org/) site.
+    - **Red error datasets**:
+        - To resolve it you might be asked to send a bug report.
```

Comment:
Here I would link to the troubleshooting FAQ, to see if the user can find the problem themselves first.

Reply:
I actually was thinking to remove the _red error datasets_ because it is covered in the troubleshooting FAQ to reduce duplicity as you suggested. But I can edit the commit if that's needed.